packages/confirm-dialog/src/vaadin-confirm-dialog-overlay.js

```diff
@@ -87,18 +89,14 @@ class ConfirmDialogOverlay extends OverlayMixin(DirMixin(ThemableMixin(PolymerEl
 customElements.define(ConfirmDialogOverlay.is, ConfirmDialogOverlay);
 
 /**
- * An extension of `<vaadin-dialog>` used internally by `<vaadin-confirm-dialog>`.
- * Not intended to be used separately.
+ * An element used internally by `<vaadin-confirm-dialog>`. Not intended to be used separately.
  * @private
  */
-class ConfirmDialogDialog extends Dialog {
+class ConfirmDialogDialog extends DialogBaseMixin(OverlayClassMixin(ThemePropertyMixin(PolymerElement))) {
```

Comment:
Compared to `Dialog`, `ElementMixin` (which also brings in `DirMixin`) is missing. Not needed in this case?

Reply:
Yes, it's not needed because `ElementMixin` is only used for public elements due to usage statistics 🙈